modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/java/com/liferay/portal/workflow/metrics/internal/search/index/TaskWorkflowMetricsIndexerImpl.java

```diff
@@ -134,6 +134,25 @@ public Document addTask(
 
 				ScriptBuilder builder = scripts.builder();
 
+				HashMapBuilder.HashMapWrapper<String, Object> hashMapWrapper =
+					HashMapBuilder.<String, Object>put(
+						"assigneeIds", assigneeIds
+					).put(
+						"assigneeType", assigneeType
+					).put(
+						"taskId", taskId
+					).put(
+						"taskName", name
+					);
+
```

Comment:
Hey @MarinhoFeliphe as you are using a HashMapBuilder here. It is possible to keep the `HashMapBuilder` in an inline declaration. To achieve this, you need to use the put option that accepts a `BaseMapBuilder.UnsafeSupplier<V, Exception> valueUnsafeSupplier` and return null when it is not a `User`.
Thx

Reply:
And keep the order here as well 👍